entries/QUnit.stack.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
+<entry type="method" name="QUnit.stack">
+	<title>QUnit.stack()</title>
+	<signature>
+		<argument name="offset" type="Number" optional="true" default="2">
+			<desc>Set the stackTrace line offset.</desc>
+		</argument>
+	</signature>
+	<desc>Return a single line source reference from the stacktrace (call stack)</desc>
+	<longdesc>
+		<p>This method returns a single line string representing the stacktrace from where it was called.</p>
+		<p>If it's given a offset argument, QUnit.stack will return the correspondent line from the call stack.</p>
+	</longdesc>
+	<example>
+		<desc>
+			The stack can be used on custom assertions and different reporters.
```

Comment:
This example doesn't make much sense, since `details.source` provides the source of the assertion. Calling `QUnit.stack` inside `log` will just provide the source of the log callback.